CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+Contributing to Opencast Studio
+===============================
+
+As an open source project, Opencast Studio welcomes contributions of many forms.
+Examples of contributions include:
+
+- Code patches
+- Documentation improvements
+- Bug reports and patch reviews
+
+
+Being Pragmatic
+---------------
+
+For every rule, there is an exception.
+If you find that there is a good reason one of the following rules does not apply to you,
+please bring it up and explain why.
+We like to be pragmatic if necessary instead of just blindly following rules.
+
+
+Provide Necessary Information
+-----------------------------
+
+If you provide a patch, please also provide an explanation of the reasoning behind this patch.
+It is much easier to understand and review code if you know its intention upfront.
```

Comment:
Should there be "," before "if you know[…]"?

Reply:
I don't think so. But I am certainly no expert on commas in the English language :grin:

Reply:
You are right. I just read through the punctuation rules ;-)